### ISSUE TITLE
Nested line item templates are missing ul and break collapse view

### ISSUE THREAD

**Issue by tobiasberge** [shopware/platform#8321]
### Shopware Version

6.7.0.0

### Affected area / extension

Platform(Default)

### Actual behaviour

The nested line-items are not nested within the collapse panel.

![Image](https://github.com/user-attachments/assets/ef21192d-ed4a-406e-a0af-06d216c4f412)

### Expected behaviour

The nested line-items should be nested within the collapse panel.

### How to reproduce

Add nested line items e.g. via app scripts.
